src/main/java/apiquality/sonar/openapi/checks/format/OAR044MediaTypeCheck.java

```diff
@@ -74,8 +81,37 @@ private void verifyMimeTypeArray(JsonNode node) {
   private void visitOpenApi3(JsonNode node) {
     if (node.getType() == OpenApi3Grammar.PARAMETER) {
       verifyParameterContent(node);
-    } else {
-      verifyContent(node);
+    }
+
+    if (node.getType() == OpenApi3Grammar.RESPONSES) {
+        List<JsonNode> responseCodes = node.properties().stream().collect(Collectors.toList());
+        for (JsonNode jsonNode : responseCodes) {
+          if (!jsonNode.key().getTokenValue().equals("204")) {
+            boolean externalRefManagement = false;
+            if (isExternalRef(jsonNode) && externalRefNode == null) {
+              externalRefNode = jsonNode;
+              externalRefManagement = true;
+            }
+            jsonNode = resolve(jsonNode);
+            verifyContent(jsonNode);
+            if (externalRefManagement) externalRefNode = null;
+          }
+        }
+    }
+
+    if (node.getType() == OpenApi3Grammar.OPERATION) {
```

Comment:
we are subscribed now to OpenApi3Grammar.PATH, why we don't have an if for that case?